docset/windows/dhcpserver/add-dhcpserverv4failover.md

```diff
@@ -73,7 +73,7 @@ This example creates a hot standby, or active-passive, failover relationship bet
 These scopes will be created on the partner DHCP server service that runs on the computer named dhcpserver2.contoso.com as part of the failover relationship creation.
 The DHCP server service that runs on the computer named dhcpserver.contoso.com will be the standby DHCP server service and the DHCP server service that runs on the computer named dhcpserver2.contoso.com will be the active DHCP server service in the failover relationship.
 
-### Example 3: Create an active-active failover relationship with specified load balance amount
+### Example 3: Create an active-active failover relationship with specified load balance amount 
```

Comment:
Looks like this is just adding trailing pace

Reply:
I think this must have already been fixed in your PR @nschonni .. I don't think we need a trailing space so will close this one. Please re-open if I am incorrect.

Reply:
Actually it looks like it was fixed in https://github.com/MicrosoftDocs/windows-powershell-docs/commit/e440a817f1798f8946ee1e6ca431b56053a9d38f but I thing this PR was just making a change to try and trigger the page to be republished

Reply:
Yes, nothing to be done on my end. added the space as I had to make a pull request. Doc site publishing was required and mentioned in the comment above.